paginate/src/main/java/com/paginate/abslistview/WrapperAdapter.java

```diff
@@ -67,6 +67,7 @@ public View getView(int position, View convertView, ViewGroup parent) {
 
     void displayLoadingRow(boolean displayLoadingRow) {
         this.displayLoadingRow = displayLoadingRow;
+        wrappedAdapter.notifyDataSetChanged();
```

Comment:
I need to call pagination.setHasMoreDataToLoad(boolean) whether the list data is null.
adapter always call onCreateViewHolder() and onBindViewHolder() while I call that.